src/wasm.h

```diff
@@ -1842,4 +1842,6 @@ template<> struct hash<wasm::Address> {
 };
 } // namespace std
 
+#include "wasm-printing.h"
+
```

Comment:
This looks like an unintentional addition.

Reply:
Oh, sorry, I meant to comment - that was intentional. I got tired of typing it in every time I debug. We do include Type, Literal, etc. printing by default, and were just missing this one, so I added it. Maybe there's a nicer place to put it though?

Reply:
Hmm, those other files don't depend on anything declared in wasm.h though. This is kind of weird in that it creates a cyclic dependency between the two files.

Reply:
Maybe a nicer way is to just embed the contents of `wasm-printing.h` in `wasm.h`? That's basically what we do for all the others. (OTOH, separating the file out is more modular)

Reply:
Oh yes, wasm-printing.h is fairly short and straightforward. I think it would make sense to put at least the declarations of the various `operator<<`s in wasm.h. It would be good to do that as a separate change, though.